.github/workflows/scan_action_logs.yml

```diff
@@ -0,0 +1,26 @@
+name: Scan Action Logs
+
+on:
+  schedule:
+    - cron: "0 5 * * *" # 5am UTC
```

Comment:
Would you be willing to change this so it does not run exactly on an hour?

Reply:
52 5 * * *